packages/mobile/src/store/offline-downloads/sagas/offlineQueueSagas/workers/downloadCollectionWorker.ts

```diff
@@ -121,14 +125,18 @@ function* downloadCollectionAsync(
   const currentUserId = yield* select(getUserId)
   if (!currentUserId) return OfflineDownloadStatus.ERROR
 
-  const apiClient = yield* getContext('apiClient')
-  const [collection] = yield* call([apiClient, apiClient.getPlaylist], {
-    playlistId: collectionId,
-    currentUserId,
-    // Needed to ensure APIClient doesn't abort when we become unreachable,
-    // allowing this job time to self-cancel
-    abortOnUnreachable: false
-  })
+  const sdk = yield* getSDK()
+  const { data = [] } = yield* call(
+    [sdk.full.playlists, sdk.full.playlists.getPlaylist],
+    {
+      playlistId: Id.parse(collectionId),
+      userId: OptionalId.parse(currentUserId)
+    }
+  )
+  const [collection] = transformAndCleanList(
```

Comment:
Do we prefer this vs `[0]` like in the other examples? (I know they were mixed before, just curious!)

Reply:
I don't have a strong preference either way. I've been trying to maintain whatever style I encounter. I personally find this style a little easier to read. But I think it's less obvious that it's taking the first item in an array if you aren't super familiar with JS syntax.